task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,7 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+ return num % 15 == 0 ? 'FizzBuzz': num % 3 == 0 ? 'Fizz': num % 5 == 0 ?'Buzz': num; 
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"